configd/src/apps/cmd/main.cpp

```diff
@@ -57,9 +58,9 @@ Cmd::finiRPC()
         _target->SubRef();
         _target = nullptr;
     }
-    if (_supervisor) {
-        _supervisor->ShutDown(true);
-        _supervisor.reset();
+    if (_server) {
+        _server.reset();
+        _supervisor = nullptr;
```

Comment:
prefer setting supervisor to nullptr first

Reply:
Fixed